src/main/java/service/WriteToFile.java

```diff
@@ -0,0 +1,5 @@
+package service;
+
+public interface WriteToFile {
```

Comment:
Incorrect interface naming. Please pay attention to it and fix all interfaces/classes names